docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

```diff
@@ -0,0 +1,24 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023
+
+On the week of November 13th 2023 we removed the option to create, configure, and delete Jira, Slack, and Webhooks integrations for your repositories. These integrations were available on the repository **Settings**, tab **Integration**.
+
+<!--TODO CY-7215 Add rationale for this removal-->
+
+## Removal of existing integrations
+
+On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->
```

Comment:
maybe @inesgomas has better visibility

we don't need an accurate date, just a ballpark that we can adjust to when the migration is code-ready

however, the expected date is a nice-to[have in this case: there are no actions to be taken, or alternatives to implement.

Reply:
I proposed to the team that tomorrow we refine those cleanups

Reply:
As @mrfyda mentioned perhaps we can release this with a timeframe of end of the year timeline and add the detail of "further comms on this will be sent via email, make sure you have them enable (with link to the email permitions page.)"

Reply:
[As agreed](https://github.com/codacy/docs/pull/1944#pullrequestreview-1754830151), we'll later add the action to re-subscribe to marketing emails.